### PR TITLE
Removes instances of Containerized Install

### PR DESCRIPTION
**Pull request by stevsmit**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1574447

Preview at https://deploy-preview-31542--osdocs.netlify.app/openshift-enterprise/latest/install/index.html

@jiajliu please review

To be merged to 3.11.

### CODE REVIEW

**Comment by jiajliu** [comment]
@jianlinliu ^

**Comment by stevsmit** [comment]
> @jianlinliu ^

Thanks! Ready for review again.

**Comment by jianlinliu** [comment]
LGTM.

File: install/index.adoc [hunk 1]
```diff
@@ -290,6 +290,11 @@ container method is used on that host. Either installation type provides the
 same functionality for the cluster, but the operating system you use determines
 how you manage services and host updates.
 
+[NOTE]
+====
+As of {product-title} 3.10, the containzerized installation method is no longer supported on Red Hat Enterprise Linux systems.
```
Comment (ahardin-rh) [inline install/index.adoc:295]: s/Red Hat Enterprise Linux/{op-system-base-full} systems

The {op-system-base-full} and other variables are defined in the modules/common-attributes.adoc file

Comment (ahardin-rh) [reply install/index.adoc:295]: Oh! Since this is 3.x docs, that variable may not exist. If it doesn't, disregard :smile:

**Pull request merged**